crates/storage/src/blueprint/sparse.rs

```diff
@@ -377,9 +380,9 @@ where
         let prev_metadata: Cow<SparseMerkleMetadata> = storage
             .storage::<Metadata>()
             .get(primary_key)?
-            .unwrap_or_default();
+            .unwrap_or(Cow::Owned(SparseMerkleMetadataV1::default().into()));
```

Comment:
You can implement `Default` for the enum; it is okay to do since we can use the latest(primary) variant of the enum if there is no entry in the storage.